### DISCUSSION TITLE
FluentValidation Blog Source Code

### DISCUSSION THREAD

**Discussion by TorreyGarland** [Megabit/Blazorise#4085] [Q&A]
Is there a gist or separate repo for the blog at https://blazorise.com/blog/reactive-ui-blazorise-fluent-validation.

Interesting article, but now comment section for the blog page.

**Comment by stsrki** [comment]
I'm not sure there is any gist for the blog. The original blog was authored by @richbryant on https://richbryant.github.io/pages/?p=/article/fluent-tinkering

---

BTW, we're currently working on incorporating fluent validation into Blazorise; see #4068, https://github.com/aladotnet/BlazoriseFluentValidation/issues/5

Expect documentation and code samples along with it.
